src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/VotingStep/partials/TextWithValue/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface TextWithValueProps {
+  children: React.ReactNode;
```

Comment:
Can you add more descriptive names for the props, but also not related to voting?